HMCL/src/main/resources/assets/lang/I18N_zh_CN.properties

```
@@ -1239,59 +1209,53 @@ update.no_browser=无法打开浏览器。网址已经复制到剪贴板，你
 update.tooltip=更新
 
 version=游戏
-version.name=游戏实例名称
-version.cannot_read=读取游戏实例失败，无法进行自动安装
-version.empty=没有游戏实例
+version.name=游戏版本名称
+version.cannot_read=读取游戏版本失败，无法进行自动安装
+version.empty=没有游戏版本
 version.empty.add=进入下载页安装游戏
-version.empty.launch=没有可启动的游戏。点击“确定”将进入“下载”页面。\n你也可以点击 HMCL 主界面左侧的“下载”按钮安装游戏，或在“实例列表”切换游戏文件夹。
+version.empty.launch=没有可启动的游戏。点击“确定”将进入“下载”页面。\n你也可以点击 HMCL 主界面左侧的“下载”按钮安装游戏，或在“版本列表”切换游戏文件夹。
 version.empty.hint=没有已安装的游戏。\n你可以切换其他游戏文件夹，或者点击此处进入游戏下载页面。
-version.game.all=全部
-version.game.april_fools=愚人节
 version.game.old=远古版
 version.game.release=正式版
 version.game.releases=正式版
 version.game.snapshot=快照
 version.game.snapshots=快照
-version.game.type=版本类型
 version.launch=启动游戏
-version.launch.empty=开始游戏
-version.launch.empty.installing=安装游戏
-version.launch.empty.tooltip=安装并启动最新正式版游戏
 version.launch.test=测试游戏
-version.switch=切换实例
+version.switch=切换版本
 version.launch_script=生成启动脚本
 version.launch_script.failed=生成启动脚本失败
 version.launch_script.save=保存启动脚本
 version.launch_script.success=启动脚本已生成完毕：%s
-version.manage=实例列表
+version.manage=版本列表
 version.manage.clean=清理游戏文件夹
 version.manage.clean.tooltip=清理“logs”和“crash-reports”文件夹
-version.manage.duplicate=复制游戏实例
+version.manage.duplicate=复制游戏版本
 version.manage.duplicate.duplicate_save=复制存档
 version.manage.duplicate.prompt=请输入新游戏名称
-version.manage.duplicate.confirm=新的游戏实例将复制该实例文件夹 (".minecraft/versions/<实例名>") 下的文件，并带有独立的运行文件夹和设置。
-version.manage.manage=实例管理
-version.manage.manage.title=实例管理 - %1s
+version.manage.duplicate.confirm=新的游戏版本将复制该版本文件夹 (".minecraft/versions/<版本名>") 下的文件，并带有独立的运行文件夹和设置。
+version.manage.manage=版本管理
+version.manage.manage.title=版本管理 - %1s
 version.manage.redownload_assets_index=更新游戏资源文件
-version.manage.remove=删除该实例
-version.manage.remove.confirm.trash=真的要删除实例“%s”吗？你可以在系统的回收站中还原“%s”文件夹来找回该实例。
-version.manage.remove.confirm.independent=由于该游戏启用了“(全局/实例特定) 游戏设置 → 版本隔离 → 各实例独立”选项，删除该实例将导致该游戏的存档等数据一同被删除！真的要删除实例“%s”吗？
+version.manage.remove=删除该版本
+version.manage.remove.confirm=真的要删除版本“%s”吗？你将无法找回被删除的文件！！！
+version.manage.remove.confirm.trash=真的要删除版本“%s”吗？你可以在系统的回收站中还原“%s”文件夹来找回该版本。
+version.manage.remove.confirm.independent=由于该游戏启用了“(全局/版本特定) 游戏设置 → 版本隔离 → 各版本独立”选项，删除该版本将导致该游戏的存档等数据一同被删除！真的要删除版本“%s”吗？
 version.manage.remove_assets=删除所有游戏资源文件
 version.manage.remove_libraries=删除所有库文件
-version.manage.rename=重命名该实例
+version.manage.rename=重命名该版本
 version.manage.rename.message=请输入要修改的名称
-version.manage.rename.fail=重命名实例失败，可能文件被占用或者名字有特殊字符。
-version.search=名称
-version.search.prompt=输入版本名称进行搜索
+version.manage.rename.fail=重命名版本失败，可能文件被占用或者名字有特殊字符。
 version.settings=游戏设置
 version.update=更新整合包
-
-warning.java_interpreted_mode=HMCL 正在运行在解释器模式的 Java 环境中，性能将会受到很大影响。\n我们建议你使用支持 JIT 的 Java 启动 HMCL 以获取最佳体验。
-warning.software_rendering=HMCL 正在使用软件渲染，性能将会受到很大影响。\n你可以访问 https://docs.hmcl.net/help.html 页面寻求帮助。
+version.pin=固定版本
+version.unpin=取消固定版本
+version.doubleClickToLaunch=双击启动该版本
+version.pinned=已固定的版本
```

HMCL 已将「版本」更名为「实例」。

- https://github.com/HMCL-dev/HMCL/issues/4374

```suggestion
version.pin=固定实例
version.unpin=取消固定实例
version.doubleClickToLaunch=双击启动该实例
version.pinned=已固定的实例
```